canvasapi/course_event.py

```diff
@@ -0,0 +1,6 @@
+from canvasapi.canvas_object import CanvasObject
+
+
+class CourseEvent(CanvasObject):
```

Comment:
Since this is a new file with a new class, ensure to add it to the documentation reference files.